pkg/util/ovs/ovs.go

```diff
@@ -18,9 +18,11 @@ type Interface interface {
 	// existed, it will be destroyed and recreated.
 	AddBridge(properties ...string) error
 
-	// DeleteBridge deletes the bridge associated with the interface. (It is an
-	// error if the bridge does not exist.)
-	DeleteBridge() error
+	// DeleteBridge deletes the bridge associated with the interface. The boolean
+	// that can be passed determines if a bridge not existing is an error. Passing
+	// true will delete bridge --if-exists, passing false will error if the bridge
+	// does not exist.
+	DeleteBridge(exists bool) error
```

Comment:
Can you change that to `ifExists`? (here and in both the real and fake implementations)